osquery/config/tests/config_tests.cpp

```diff
@@ -121,7 +121,11 @@ class TestConfigPlugin : public ConfigPlugin {
 
     std::string content;
     auto s = readFile(getTestConfigDirectory() / "test_noninline_packs.conf",
-                      content);
+                      content,
+                      0,
```

Comment:
@farfella can you elaborate more on this change? If there's an inconsistency with the `read` API we should fix that.

Reply:
It's related to this: https://github.com/osquery/osquery/issues/5661

I'm though preparing a PR to fix at least part of the issue (the most impacting), so that it won't fail again.
Then refactors can come later.

Reply:
What @Smjert said. :-) We had a discussion last night on slack windows channel how the overlapped io implementation has a couple of bugs. So, I set these couple of test cases to call readFile with blocking semantics, since the default for readFile is non-blocking and it causes these tests to pass/fail in a non-deterministic fashion.

Reply:
Makes sense, do you mind clarifying if the recent fix to overlapped IO also needs these fixes here? I’d rather avoid blocking here unless we plan to block in all tests.

Reply:
The PR I've done should fix this issue and so it shouldn't require this fix.

Reply:
Thanks @Smjert! I will revert these cases.